kafka-merge-pr.py

```diff
@@ -72,7 +72,7 @@
 
 DEV_BRANCH_NAME = "trunk"
 
-DEFAULT_FIX_VERSION = os.environ.get("DEFAULT_FIX_VERSION", "0.10.0.0")
+DEFAULT_FIX_VERSION = os.environ.get("DEFAULT_FIX_VERSION", "0.10.0.1")
```

Comment:
If someone runs kafka-merge-py from trunk, this doesn't matter at all :)
If they run it from here, hopefully its because they are working on 0.10.0 specific change, and they'll get a reasonable version...